code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -198,8 +198,9 @@
 	..()
 	new /obj/item/weapon/grenade/empgrenade(src)
 	new /obj/item/weapon/grenade/empgrenade(src)
+	new /obj/item/weapon/grenade/empgrenade(src)
+	new /obj/item/weapon/grenade/empgrenade(src)
 	new /obj/item/weapon/implanter/emp(src)
-	new /obj/item/device/flashlight/emp(src)

```

Comment:
No. That's not safe to ignore. You need to make sure your master is up to date before editing files.